pymc/distributions/distribution.py

```diff
@@ -589,18 +582,11 @@ def dist(
         # Create the RV with a `size` right away.
         # This is not necessarily the final result.
         graph = cls.rv_op(*dist_params, size=create_size, **kwargs)
-        graph = maybe_resize(
-            graph,
-            cls.rv_op,
-            dist_params,
-            ndim_expected,
-            ndim_batch,
-            ndim_supp,
-            shape,
-            size,
-            change_rv_size_fn=cls.change_size,
-            **kwargs,
-        )
+
+        # Replicate dimensions may be prepended via a shape with Ellipsis as the last element:
+        if shape is not None and Ellipsis in shape:
+            replicate_shape = cast(StrongShape, shape[:-1])
+            graph = cls.change_size(rv_var=graph, new_size=replicate_shape, expand=True)
```

Comment:
I don't think I've implemented the expand kwarg in censored... but should

```suggestion
            graph = cls.change_size(rv_var=graph, new_size=replicate_shape)
```

Reply:
https://github.com/pymc-devs/pymc/blob/a3bab7ddc0f4612ad039a7d2755f22fa6c8c0a72/pymc/distributions/censored.py#L103

Reply:
is it gonna work at all then?

Reply:
right now it's working if not "expanding". We can put a `NotImplementedError` if expand is `True` in `Censored.change_size` for the time being

Reply:
You wanna do that commit?
I gotta go run t the supermarket before the vegetables are gone 🙂

Reply:
Fixed